src/pivot-table/hooks/use-column-width.ts

```diff
@@ -158,6 +155,12 @@ export default function useColumnWidth(
               cellWidth = getColumnWidth(columnWidth, fitToContentWidth);
             }
 
+            // The last cell setting should override the other cells in that column, so we don't pick the max
+            const isTypeAuto = !header.columnWidth || header.columnWidth.type !== ColumnWidthType.Auto;
```

Comment:
Seems like this variable should be called `isNotAuto`?

Reply:
this was incorrect, which also caused the rendering test failure